app/views/wallaby/resources/form/_float.html.erb

```diff
@@ -11,7 +11,7 @@ The following params are the variables that can be used in this partial.
   <%= form.label field_name, metadata[:label] %>
   <div class="row">
     <div class="col-xs-6 col-sm-4">
-      <%= form.number_field field_name, class: 'form-control' %>
+      <%= form.number_field field_name, class: 'form-control', step: metadata[:step] || 0.1 %>
```

Comment:
passing in an option will be better i think

```
<%= form.number_field field_name, class: 'form-control', metadata[:options] || {} %>
```

and not sure if step default to 0.1 is a common practice

Reply:
@tian-im updated :)